report.h

```diff
@@ -63,6 +63,10 @@
 #define MESSAGE_ENABLED 4
 #define MESSAGE_DISABLED 5
 
+// Variable for holding voltage value after ADC
+extern uint32_t voltage_result[5];
```

Comment:
You've introduced a bunch of magic numbers in here. Instead, make them named constants.

```c
#define VOLTAGE_CHANNEL_COUNT 5
extern uint32_t voltage_result[VOLTAGE_CHANNEL_COUNT];
```

and in report.c: 

```c
for (i = 0; i<VOLTAGE_CHANNEL_COUNT; i++){
    printInteger(voltage_result[i]);
    if (i<VOLTAGE_CHANNEL_COUNT - 1)
    printPgmString(PSTR(","));
}
```